exec/cnex/module.c

```diff
@@ -1,16 +1,30 @@
-#include "module.h"
+#ifdef _WIN32
+#define _CRT_SECURE_NO_WARNINGS
+#else
+// The following is needed for strdup on Posix platforms.
+#define _POSIX_C_SOURCE 200809L
+#endif
 
 #include <errno.h>
 #include <stdio.h>
 #include <string.h>
 
+#ifdef _WIN32
+#include <io.h>
+#define strdup _strdup
+#else
+#include <unistd.h>
+#endif
+
 #include "bytecode.h"
 #include "cell.h"
 #include "exec.h"
+#include "module.h"
 #include "nstring.h"
 #include "support.h"
 
 
+
```

Comment:
Why adding extra blank line here? There were already two, you surely don't need three.

Reply:
Fixed.